examples/hosts/app-integration/external-data/src/modelInterfaces.ts

```diff
@@ -17,6 +17,11 @@ export interface IAppModel extends IEventProvider<IAppModelEvents> {
      * A task tracker list.
      */
     readonly taskList: ITaskList;
+
+    /**
+    * Trigger custom signals for external data change and for sending the data
+    */
+    readonly debugSendCustomSignal: () => void;
```

Comment:
Adding this directly on the IAppModel is prob fine for now -- if we add many debug things we may want to establish a separate debug object (e.g. IAppModelDebug) and hang the debug functionality on that.

Reply:
Understood. Yeah the next thing I was going to build was a new debug signal for fetching the data potentially. The second debug method probably warrants a new class of it's own!